src/client/transfer_actor/mod.rs

```diff
@@ -277,23 +277,12 @@ impl Client {
                                         validation.clone(),
                                     ))?;
                                     info!("Transfer successfully validated.");
-                                    if let Some(tap) = validation.proof {
+                                    if let Some(dap) = validation.proof {
```

Comment:
```suggestion
                                    if let Some(tap) = validation.proof {
```

just to update terminology, transfer agreement proof